terraform/lambda/variables.tf

```diff
@@ -19,3 +19,8 @@ variable "environment_variables" {
     EMPTY_VARIABLE = ""
   }
 }
+
+variable "timeout" {
+  description = "The amount of time your Lambda Function has to run in seconds. Defaults to 3"
```

Comment:
nit: "Function" doesn't need to be capitalised. Don't repeat the default in the description.